### PR TITLE
chore: dynamic plugins update for 1.1...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: dynamic plugins update for 1.1 (RHIDP-1475)

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc
Comment (nickboldt) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:106]: this was already in the Community support table on release-notes/con-relnotes-techpreview-features.adoc and it's third party owned (not RH) so it should be community according to Christophe.

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc
Comment (nickboldt) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:296]: plugin ID is changed since RHDH 1.0.

**Comment by Gerry-Forde** [comment]
@nickboldt This looks good to me. Before we merge the PR should we wait for @kadel to reply regarding the missing **Required Variables** values referred to in https://issues.redhat.com/browse/RHIDP-1475

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (davidfestal) [inline release-notes/con-relnotes-techpreview-features.adoc:66]: For this one, the require_valiables is: `BITBUCKET_WORKSPACE`


cf https://github.com/janus-idp/backstage-showcase/blob/1.1.x/dynamic-plugins.default.yaml#L604

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (davidfestal) [inline release-notes/con-relnotes-techpreview-features.adoc:70]: For this one the required_variables is: `BITBUCKET_WORKSPACE`

cf https://github.com/janus-idp/backstage-showcase/blob/1.1.x/dynamic-plugins.default.yaml#L622

**Review by davidfestal — CHANGES_REQUESTED** [review]

**Comment by davidfestal** [comment]
cc @Gerry-Forde @kadel

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (nickboldt) [reply release-notes/con-relnotes-techpreview-features.adoc:66]: please use the `suggestion`  feature in GH to make it easier to just push your change into my PR. 

image

(or ctrl-g)

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc
Comment (nickboldt) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:116]:
```suggestion
|Bitbucket |Backend |@backstage/plugin-catalog-backend-module-bitbucket-cloud |A Backstage catalog backend module that helps integrate towards Bitbucket Cloud. |0.1.28 |Community Support |dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-bitbucket-cloud |`BITBUCKET_WORKSPACE` |Disabled
```

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc
Comment (nickboldt) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:118]:
```suggestion
|Bitbucket |Backend |@backstage/plugin-catalog-backend-module-bitbucket-server |A Backstage catalog backend module that helps integrate towards Bitbucket Server. |0.1.26 |Community Support |dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-bitbucket-server |`BITBUCKET_WORKSPACE` |Disabled
```

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (nickboldt) [reply release-notes/con-relnotes-techpreview-features.adoc:66]: is this what you meant to suggest? https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/75/files#r1525362220 for both the Cloud and Server `plugin-catalog-backend-module-bitbucket-*` entries in the ref guide table?  What about the scaffolders?

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (davidfestal) [reply release-notes/con-relnotes-techpreview-features.adoc:70]: There was a copy/paste error here: it is `BITBUCKET_HOST`

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (davidfestal) [reply release-notes/con-relnotes-techpreview-features.adoc:66]: For cloud yes, for server it is `BITBUCKET_HOST` (see my last comment below)

Comment (nickboldt) [reply release-notes/con-relnotes-techpreview-features.adoc:66]: ... you're still not using the suggest feature. :D

File: release-notes/con-relnotes-techpreview-features.adoc
Comment (nickboldt) [reply release-notes/con-relnotes-techpreview-features.adoc:70]: fixed in suggestions elsewhere in this PR. Please review and merge the suggestions if they're correct.

**Comment by Gerry-Forde** [comment]
@nickboldt It looks like we just need to commit those couple of updates for the required variables and then we're good to merge this. Is there anything else required?

**Comment by nickboldt** [comment]
> It looks like we just need to commit those couple of updates for the required variables and then we're good to merge this. Is there anything else required?

I was waiting for @davidfestal and @kadel to confirm that there were no other env vars we needed to list, but I'll set this to automerge and we can iterate on Monday if there's more to be done.

**Comment by Gerry-Forde** [comment]
@nickboldt 

- The table of dynamic plugins looks good to me.
- WRT the updated Release Notes content, I understand that the tables of plugins have been removed from the RNs for RHDH 1.1 - @shipsing can you confirm, please?

**Comment by shipsing** [comment]
> @nickboldt
> 
> * The table of dynamic plugins looks good to me.
> * WRT the updated Release Notes content, I understand that the tables of plugins have been removed from the RNs for RHDH 1.1 - @shipsing can you confirm, please?

Hi @Gerry-Forde @nickboldt, Yes, I have removed the table of plugins  from RN to avoid redundnat work. We have all the plugin details in the admin guide and we have provided a reference to that in our 1.1 RN. 

See this section: https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-79/rel-notes-rhdh/#_plugins-available-in-red-hat-developer-hub

**Review by davidfestal — APPROVED** [review]

**Review by hmanwani-rh — APPROVED** [review]

**Pull request merged**
